### PR TITLE
Redesign Implementation Header & Footer

### PR DESCRIPTION
**Pull request by katibest**
## Motivation
Wanted to Implement the new header and footer for the redesign. 
Solved #305 

## Changelog
Updated 
* Navigation
* Footer 
* Refactored layout
* Removed code (html and css) from previous designs

## Screenshot
screen shot 2018-06-21 at 11 01 04 am

### CODE REVIEW

File: apps/explorer_web/lib/explorer_web/templates/layout/_topnav.html.eex [hunk 1]
```diff
@@ -1,36 +1,56 @@
...
+        <button class="btn btn-outline-success my-2 my-sm-0 sr-only" type="submit">Search</button>
       <% end %>
+      <ul class="navbar-nav">
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            Mainnet
+          </a>
+          <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+            <a class="dropdown-item" href="https://explorer-sokol.poanetwork/">POA Sokol</a>
+            <a class="dropdown-item" href="https://explorer-core.poanetwork/">POA Core</a>
```
Comment (Stamates) [inline apps/explorer_web/lib/explorer_web/templates/layout/_topnav.html.eex]: Need to use gettext() for Mainnet (line 46), POA Sokol (line 49), and POA Core (line 50)

File: apps/explorer_web/lib/explorer_web/templates/layout/_footer.html.eex [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+        <i class="fab fa-instagram"></i>
+      </a>
+      <a href="https://www.twitter.com/PoaNetwork/" target="_blank" data-toggle="tooltip" data-placement="top" title="Twitter">
+        <i class="fab fa-twitter"></i>
+      </a>
+      <a href="https://www.twitter.com/PoaNetwork/" target="_blank" data-toggle="tooltip" data-placement="top" title="Telegram">
+        <i class="fab fa-telegram-plane"></i>
+      </a>
+    </div>
+
```
Comment (Stamates) [inline apps/explorer_web/lib/explorer_web/templates/layout/_footer.html.eex]: Maybe use gettext() for social link titles?

Comment (katibest) [reply apps/explorer_web/lib/explorer_web/templates/layout/_footer.html.eex]: Updated!

File: apps/explorer_web/lib/explorer_web/templates/layout/_topnav.html.eex (hunk 1 above)
Comment (katibest) [reply apps/explorer_web/lib/explorer_web/templates/layout/_topnav.html.eex]: Updated!

**Pull request merged**